homeassistant/components/fireservicerota/sensor.py

```diff
@@ -0,0 +1,154 @@
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import ATTRIBUTION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up FireServiceRota sensor based on a config entry."""
+    data = hass.data[DOMAIN][entry.entry_id]
+    entry_id = entry.entry_id
+    unique_id = entry.unique_id
+
+    async_add_entities(
+        [IncidentsSensor(data, entry_id, unique_id, "Incidents", "mdi:fire-truck")],
+        True,
+    )
+
+
+class IncidentsSensor(RestoreEntity):
+    """Representation of FireServiceRota incidents sensor."""
+
+    def __init__(self, data, entry_id, unique_id, name, icon):
+        """Initialize."""
+        self._data = data
+        self._entry_id = entry_id
+        self._unique_id = unique_id
+        self._name = name
+        self._icon = icon
+
+        self._enabled_default = True
+        self._available = True
+        self._device_class = None
+
+        self._state = None
+        self._state_attributes = {}
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def icon(self) -> str:
+        """Return the icon to use in the frontend."""
+        return self._icon
+
+    @property
+    def state(self) -> str:
+        """Return the state of the sensor."""
+        return self._state
+
+    @property
+    def unique_id(self) -> str:
+        """Return the unique ID for this sensor."""
+        return f"{self._unique_id}_{self._name}"
+
+    @property
+    def device_state_attributes(self) -> object:
+        """Return available attributes for sensor."""
+        attr = {}
+        data = self._state_attributes
+
+        if data:
+            for value in (
+                "trigger",
+                "created_at",
+                "message_to_speech_url",
+                "prio",
+                "type",
+                "responder_mode",
+                "can_respond_until",
+            ):
+                if data.get(value):
+                    attr[value] = data[value]
+
+            try:
+                for address_value in (
+                    "latitude",
+                    "longitude",
+                    "address_type",
+                    "formatted_address",
+                ):
+                    attr[address_value] = data.get("address").get(address_value)
+            except (KeyError, AttributeError):
+                pass
+
+            attr[ATTR_ATTRIBUTION] = ATTRIBUTION
+            return attr
+
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        return self._enabled_default
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._available
```

Comment:
This is always true
```suggestion
        return True
```

Reply:
Already fixed with inheritance of Entity helper